test_lib/compaction.py

```diff
@@ -23,6 +32,36 @@ def from_str(cls, output_str):
             raise ValueError(err_msg) from attr_err
 
 
+def get_gc_mode(node: BaseNode, keyspace: str, table: str) -> str | GcMode:
+    """Get a given table GC mode
+
+    :Arguments:
+        node
+        keyspace
+        table
```

Comment:
Please link here the issue that made you do the workaround.

Reply:
@roydahan , it is found in line 51.